internal/repository/repository.go

```diff
@@ -0,0 +1,68 @@
+package repository
+
+import (
+	"github.com/VTerenya/employees/internal"
+	"strconv"
+)
+
+type RepositoryHandler interface {
+	GetPositions() map[string]internal.Position
+	GetEmployees() map[string]internal.Employee
+	AddPosition(p internal.Position)
+	AddEmployee(e internal.Employee)
+}
+
+type Repository struct {
+	RepositoryHandler
+}
+
+func NewRepository(data *database) *Repository {
+	return &Repository{
+		NewTodoRepository(data),
+	}
+}
+
+type todoRepository struct {
```

Comment:
Why do you need two structs here? I guess just  `Repository` struct should be enough.